marshmallow-metaclass.py

```diff
@@ -0,0 +1,82 @@
+from marshmallow import Schema, fields
+
+
+class Meta(type):
+    """The meta class."""
+
+    def __new__(cls, name, bases, attrs):
+        """Custom __new__  method."""
+
+        def __init__(self, **kwargs):
+            """Custom __init__ method."""
+            # get all marshmallow fields
+            self.fields = []
+            for field in dir(self):
+                f = getattr(self, field)
+                if hasattr(f, '__module__') \
+                        and f.__module__ == 'marshmallow.fields':
+                    self.fields.append(field)
+
+            # create schema
+            self.field_keys = dict()
+            for field in self.fields:
+                self.field_keys[field] = getattr(self, field)
+            self.schema = Schema.from_dict(self.field_keys)()
+
+            # desirialize
+            self.data = self.schema.load(kwargs)
+
+            # set attributes
+            for key, val in self.data.items():
+                setattr(self, key, val)
+            for field in self.fields:
+                if not field in self.data.keys():
+                    setattr(self, field, None)
+
+        def asdict(self):
+            """Return dictionary."""
+            dictionary = self.data
+            for field in self.fields:
+                if not field in self.data.keys():
+                    dictionary[field] = None
+            return dictionary
+
+        # create new class and add custom __init__ method
+        new_class = super().__new__(cls, name, bases, attrs)
+        setattr(new_class, '__init__', __init__)
```

Comment:
I think this means if I define my own `__init__` method in my class, this will overwrite that `__init__` method.

Reply:
I also added an example where the `__init__` method is overridden. The `super().__init__(**kwargs)` works just fine. I hope it's fine for you as well

Reply:
Oh, I just found out that when you write your won `__init__`, it's not used

Reply:
I've made a fix for this already. I'll push the code later because I'm still editing the `README.md`